Source/WebCore/testing/Internals.idl

```diff
@@ -684,7 +684,7 @@ typedef (FetchRequest or FetchResponse) FetchObject;
 
     unsigned long long layerIDForElement(Element element);
     
-    unsigned long long scrollingNodeIDForNode(optional Node? node = null);
+    sequence<unsigned long long> scrollingNodeIDForNode(optional Node? node = null);
```

Comment:
It's pretty unclear that the result here is a pair of processID, nodeID; here it looks more like a sequence of nodes.

Reply:
I'm not sure there's a great way to specify a pair here.